README.md

```diff
@@ -1,6 +1,6 @@
 # Offst
 
-[![Build Status](https://travis-ci.com/freedomlayer/offst.svg?branch=master)](https://travis-ci.com/freedomlayer/offst)
+[![Build Status](https://travis-ci.com/freedomlayer/offst.svg?branch=master)](https://travis-ci.com/freedomlayer/offst/branches)
```

Comment:
@pzmarzly : I think we should find out the root cause for why `?branch=master` doesn't give the desired result. Maybe we can ask on the travis forum?

Reply:
`?branch=master` does show status for master. It's just that for whatever reason Travis website by default shows the latest build, not necessarily on master

Reply:
If we think this is a bug in Travis, maybe we should report it.